homeassistant/components/wittiot/manifest.json

```diff
@@ -0,0 +1,9 @@
+{
+  "domain": "wittiot",
+  "name": "WittIOT",
+  "codeowners": ["@Wittiot"],
+  "config_flow": true,
```

Comment:
I believe you are setting up a single device, so you should set ` "integration_type": "device",`